README.md

```diff
@@ -1,4 +1,4 @@
-Cdn77 Coding Standard [![Build Status](https://travis-ci.org/cdn77/coding-standard.svg)](https://travis-ci.org/cdn77/coding-standard)
+Cdn77 Coding Standard [![Build Status](https://travis-ci.org/cdn77/coding-standard.svg?branch=master)](https://travis-ci.org/cdn77/coding-standard)
```

Comment:
👍 noticed this thing recently, very weird from travis

Reply:
yep, doesn't make sense really